linkis-public-enhancements/linkis-udf/linkis-udf-service/src/main/java/org/apache/linkis/udf/api/UDFRestfulApi.java

```diff
@@ -217,9 +217,7 @@ public Message addUDF(HttpServletRequest req, @RequestBody JsonNode json) {
       udfvo.setCreateUser(userName);
       udfvo.setCreateTime(new Date());
       udfvo.setUpdateTime(new Date());
-      udfService.addUDF(udfvo, userName);
-      message = Message.ok();
-      //            message.data("udf", udfvo);
+      message = Message.ok().data("udfId", udfService.addUDF(udfvo, userName));
```

Comment:
This changes the structure of the result set

udfService.addUDF(udfvo, userName) should return the the entity